testassets/InteropTestsClient/InteropTestsClient.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
qq: is switching the  Grpc.Core based interop client to net5.0 required? (I'm assuming this is done for consistency?)

Reply:
It's just done for consistency.